### PR TITLE
templates: display more data for the possible matches result

### PR DESCRIPTION
**Pull request by Dinika**
localhost_5000_holdingpen_list_
Search results view when multiple recoords match

localhost_5000_holdingpen_list__page 1 size 10 2
Search results view when only 1 record matched

localhost_5000_holdingpen_21
Detailed view

Signed-off-by: Dinika Saxena <dinika.saxena@cern.ch>

### CODE REVIEW

File: src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html [hunk 1]
```diff
@@ -293,12 +293,20 @@ <h5>MATCHES</h5>
...
                   <div class="panel-body">
-                    <p ng-if="match.authors" class="authors">
-                      <span ng-repeat="author in match.authors">{{author.full_name}}
-                        <span ng-if="!$last"> ; </span>
-                      </span>
-                    </p>
+                    <div ng-if="match.authors" class="authors">
+                      <span ng-repeat="author in match.authors">{{author.full_name}} <span ng-if="!$last"> ; </span> </span>
+                    </div>
+                    <ul class="list-inline">
```
Comment (harunurhan) [inline src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html]: this `ul` will show up no matter, how does it look when both `arxiv_eprint` and `number_of_pages` are missing?

File: src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html [hunk 2]
```diff
@@ -222,10 +222,20 @@ <h5>MATCHES</h5>
                     </div>
                     <div id="#match-accordion-item-{{record._id}}-{{match.control_number}}" class="panel-collapse collapse" ng-class="{ 'in': match.control_number == selectedBestMatchesById[record._id] }">
                         <div class="panel-body">
-                          <p ng-if="match.authors" class="authors">
+                          <div ng-if="match.authors" class="authors">
                             <span ng-repeat="author in match.authors">{{author.full_name}} <span ng-if="!$last"> ; </span> </span>
-                          </p>
+                          </div>
+                          <ul class="list-inline">
```
Comment (harunurhan) [inline src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html]: ditto

File: src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html [hunk 3]
```diff
@@ -292,14 +293,46 @@ <h5>MATCHES</h5>
...
+                  <div class="panel-body matches">
+                    <div ng-if="match.authors" class="authors">
+                      <span ng-repeat="author in match.authors">{{author.full_name}} <span ng-if="!$last"> ; </span></span>
+                      ({{vm.record.metadata.authors.length}} Authors)
+                    </div>
+                    <dl class="dl-horizontal">
+                      <dt ng-if="match.publication_info">Published In</dt>
+                      <dd>
+                        <span ng-repeat="pub_info in match.publication_info">
+                          <span ng-if="pub_info.journal_title">{{pub_info.journal_title}}</span>
```
Comment (harunurhan) [inline src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html:305]: maybe be here we can do another `ng-repeat` on keys of `pub_info`.

File: src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html [hunk 4]
```diff
@@ -222,10 +222,47 @@ <h5>MATCHES</h5>
...
-                          <p ng-if="match.abstract">{{match.abstract}}</p>
+                            <span ng-if="match.authors_count">({{match.authors_count}} Authors)</span>
+                          </div>
+
+                          <dl class="dl-horizontal">
+
+                            <dt ng-if="match.publication_info">Published In</dt>
+                            <dd>
+                              <span ng-repeat="pub_info in match.publication_info">
+                                <span ng-if="pub_info.journal_title">{{pub_info.journal_title}}</span>
```
Comment (harunurhan) [inline src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html:235]: ditto

File: src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results_details_hep.html [hunk 5]
```diff
@@ -34,10 +34,38 @@ <h4 class="custom-h">
 <div class="authors" ng-if="record._source.metadata.authors">
     <span ng-repeat="author in record._source.metadata.authors | limitTo:10">
       {{author.full_name}}<span ng-if="!$last">; </span><span
-      ng-if="$last && record._source.metadata.authors.length > 10">; et al. ({{ record._source.metadata.authors.length-10 }} more authors)</span>
+      ng-if="$last && record._source.metadata.authors.length > 10">; et al. ({{ record._source.metadata.authors.length }} Authors)</span>
     </span>
 </div>
 
+<div class="metadata">
```
Comment (harunurhan) [inline src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results_details_hep.html]: this is not related to matches?

File: src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html (hunk 3 above)
Comment (Dinika) [reply src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html:305]: There are two issues with that approach:
- I want to display the publication-info in a certain sequence (first the journal_title, then journal_volume, year, and so on). I am not sure if this can be done using ng-repeat

- There's a field in `pub_info` (`pubinfo_freetext`) that is not needed to be displayed

That said, I agree that the current solution might not be the most elegant

Comment (harunurhan) [reply src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html:305]: ok, as long as you don't display all, this approach is OK.

**Review by harunurhan — APPROVED** [review]
